code/modules/clothing/armor/leather.dm

```diff
@@ -27,19 +27,19 @@
 	max_integrity = 350
 	body_parts_covered = CHEST|GROIN|VITALS|LEGS|ARMS
 	prevent_crits = list(BCLASS_CUT, BCLASS_BLUNT, BCLASS_TWIST)
-	armor = list("blunt" = 75, "slash" = 60, "stab" = 30, "piercing" = 10, "fire" = 0, "acid" = 0)
+	armor = ARMOR_LEATHER_GOOD
```

Comment:
Good call on the armor values for just a fucking coat but OH MY GOD WHO GAVE THIS BLUNT CRIT RESISTANCE AS IF IT WERE A METAL ARMOR NOOOOOOOOOOOOOOO